src/clients/FtpGateway.cc

```diff
@@ -2608,7 +2608,7 @@ Ftp::Gateway::appendSuccessHeader()
          * not be seeing this condition any more because we'll only
          * send REST if we know the theSize and if it is less than theSize.
          */
-        debugs(0,DBG_CRITICAL,HERE << "Whoops! " <<
+        debugs(0,DBG_CRITICAL,"Whoops! " <<
```

Comment:
This PR automatically removes HERE, and nothing else should be required. Other issues are covered other comments, so I will not repeat those arguments here.